src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestV4ServiceImpl.java

```diff
@@ -186,13 +186,15 @@ protected ConcurEventNotificationProcessingResultsDTO processTravelRequest(Strin
         }
         
         String reportNumber = requestAsListItem.getRequestId();
-        String travelerName = String.join(StringUtils.SPACE, requestAsListItem.getOwner().getFirstName(), 
+        String reportName = requestAsListItem.getName();
+        String reportStatus = requestAsListItem.getApprovalStatus() != null   ? requestAsListItem.getApprovalStatus().getName() : CUKFSConstants.NULL;
+        String travelerName = String.join(StringUtils.SPACE, requestAsListItem.getOwner().getFirstName(),
```

Comment:
@jhulslander What property should I use to get the Request Name? Currently I am using getName. I think the travelerName was already in the report (as owner).

Reply:
Here is a link to the existing report: https://kfs-test-reports.kuali.cornell.edu/cncr/Concur_Event_Notification_V2_Processing_Results_20230110-08-16-55-167.txt .... yes  the name is already displayed.  Let me look to see if I can find if it is mapped to a DTO

Reply:
ConcurExpenseV3ListItemDTO already has a variable mapped to "name"

ConcurRequestV4ListItemDTO already has a variable mapped to "name"

both appear to be the report name and not the traveler name

Reply:
OK, I think I was getting confused, let me go back and start over again